sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSourceSuite.scala

```diff
@@ -341,6 +347,33 @@ class FileStreamSourceSuite extends FileStreamSourceTest with SharedSQLContext {
     Utils.deleteRecursively(tmp)
   }
 
+  test("metadataPath should be in checkpointLocation") {
```

Comment:
I want to check the FileStreamSource.metadataPath value. Let me just make it public to avoid the reflection.